dashboards/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from openstates.data.models import LegislativeSession
+
+
+class DataQualityDashboard(models.Model):
+
+    state = models.CharField(max_length=3)
+    chamber = models.CharField(max_length=20)
+    session = models.CharField(max_length=20)
```

Comment:
let's use a FK to session here, which means you can drop state/session char fields

Reply:
Fixed.